addon/templates/components/html-select.hbs

```diff
@@ -1,4 +1,4 @@
-<select {{action 'change' on='change'}} class="form-control">
+<select {{action 'change' on='change'}} class="{{mainComponent.elementClass}}">
```

Comment:
Well I guess as this module is always utilizing bootstrap I will switch to the suggested forced form-control class element.

Yes can do the docs and examples, but not yet, currently working on other things. Maybe this evening or tomorrow.